Source/MLXNN/Quantized.swift

```diff
@@ -121,14 +130,19 @@ open class QuantizedEmbedding: Embedding {
 ///
 /// ### See Also
 /// - ``init(weight:bias:groupSize:bits:)``
-open class QuantizedLinear: Linear {
+open class QuantizedLinear: Linear, Quantized {
 
     public let groupSize: Int
     public let bits: Int
 
     public let scales: MLXArray
     public let biases: MLXArray
 
+    open override var shape: (Int, Int) {
+        let shape = weight.shape2
+        return (shape.0, shape.1 * 32 / bits)
+    }
```

Comment:
This is also a breaking change but IMHO it was broken before (returning the size of the quantized arrays which was useless)